test/legacy/roles/test_digital_ocean/tasks/project_facts.yml

```diff
@@ -0,0 +1,55 @@
+---
+- set_fact:
+    proj:
+      - AnsibleDev1
+      - AnsibleDev2
+
+# - name: Create new projects
```

Comment:
Does this fail?

Reply:
It works. The reason the test is commented out is because the API doesn't allow for deleting projects, only creating. So the test cannot execute that step without manually deleting if it isn't the first execution.